scheduler/src/cook/kubernetes/api.clj

```diff
@@ -174,12 +174,14 @@
          (util/make-nested-atom-updater node-name->pod-name->pod pod->node-name get-pod-namespaced-key)
          (partial cook-pod-callback-wrap cook-pod-callback compute-cluster-name)] ; Invoke the cook-pod-callback if its a cook pod.
         old-all-pods @all-pods-atom]
-    (log/info "In" compute-cluster-name "compute cluster, pod watch processing pods:" (keys namespaced-pod-name->pod))
+    (log/info "In" compute-cluster-name "compute cluster, pod watch processing pods"
```

Comment:
Keep this at the full list. Its only once an hour and gives us a solid ground truth of what pods are in the system, so we can determine if there is an inconsistency. (This was extremely helpful for me to diagnose the cluster rename issue.)

Do split into one line for each pod, instead of one huge line with all pods.

Reply:
Sounds good, done

Reply:
I think we want different log lines. Newlines are seen by us in the log, but doing it as lots of separate log lines avoids creating a single huge unwieldy message.

Reply:
Made the changes we discussed offline.